### PR TITLE
staging/publishing: skip unsupported 1.10 branch

### PR DESCRIPTION
**Pull request by sttts**
According to https://github.com/kubernetes/client-go/blob/master/README.md, 1.10 is out of support.

### CODE REVIEW

**Comment by nikhita** [comment]
/sig release
/area release-eng
/kind cleanup
/release-note-none

/cc @dims @liggitt 

/hold
Ref: https://github.com/kubernetes/kubernetes/pull/73023#issuecomment-456910735

**Comment by liggitt** [comment]
final 1.10 release was cut and has been published. are these files net new?

File: staging/publishing/rules.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+skip-source-branches:
+- release-1.10
```
Comment (liggitt) [inline staging/publishing/rules.yaml]: can we delete all the 1.10 rules?

Comment (nikhita) [reply staging/publishing/rules.yaml]: Yup. :+1: 

> final 1.10 release was cut and has been published. are these files net new?

Only the last commit is new.

**Comment by nikhita** [comment]
Created https://github.com/kubernetes/kubernetes/pull/74115 to remove the 1.10 rules since this PR is having trouble detecting the cncf-cla label.

@BenTheElder @amwat another check-cla bug :(

/close

**Comment by BenTheElder** [comment]
we're pretty sure this is a bug in the CLA bot reporting to the wrong
commit.

**Pull request closed without merging**